acceptance/openstack/dataarts/v1.1/clusters_test.go

```diff
@@ -18,19 +18,19 @@ func TestDataArtsClusterLifecycle(t *testing.T) {
 	}
 
 	vpcID := clients.EnvOS.GetEnv("VPC_ID")
-	subnetID := clients.EnvOS.GetEnv("NETWORK_ID")
+	subnetID := clients.EnvOS.GetEnv("SUBNET_ID")
 	secGroupId := clients.EnvOS.GetEnv("SECURITY_GROUP_ID")
 
 	client, err := clients.NewDataArtsV11Client()
 	th.AssertNoErr(t, err)
 
 	dataStore := cluster.Datastore{
 		Type:    "cdm",
-		Version: "2.9.1.200",
+		Version: "2.10.0.100",
 	}
 	instance := cluster.Instance{
 		AZ:        "eu-de-01",
-		FlavorRef: "a79fd5ae-1833-448a-88e8-3ea2b913e1f6",
+		FlavorRef: "5ddb1071-c5d7-40e0-a874-8a032e81a697",
```

Comment:
there is no api for obtain this flavor id?

Reply:
Unfortunately, not. They are hardcoded in the documentation.